web/annotation_editor_layer_builder.js

```diff
@@ -91,10 +89,9 @@ class AnnotationEditorLayerBuilder {
     // Create an AnnotationEditor layer div
     const div = (this.div = document.createElement("div"));
     div.className = "annotationEditorLayer";
-    div.tabIndex = 0;
     div.hidden = true;
     div.dir = this.#uiManager.direction;
-    this.pageDiv.append(div);
+    view.addLayer(div, "annotationEditorLayer");
```

Comment:
Here, and elsewhere, I'm not crazy about this implementation. Can we please follow the pattern from the textLayer instead?
See
https://github.com/mozilla/pdf.js/blob/e647311a89ca9e5c69e4ccac07b1fed0f88bc9b8/web/text_layer_builder.js#L61-L65
and
https://github.com/mozilla/pdf.js/blob/e647311a89ca9e5c69e4ccac07b1fed0f88bc9b8/web/pdf_page_view.js#L895-L900
Hence this would become
```suggestion
    this.onAppend(div);
```
with the following added *just after* [this code](https://github.com/mozilla/pdf.js/blob/e647311a89ca9e5c69e4ccac07b1fed0f88bc9b8/web/pdf_page_view.js#L1052)
```js
this.annotationEditorLayer.onAppend = annotationEditorLayerDiv => {
  this.addLayer(annotationEditorLayerDiv, "annotationEditorLayer");
};
```